content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -0,0 +1,24 @@
+import {connect} from "react-redux";
+import React from "react";
+
+export class _DiscoveryStreamBase extends React.PureComponent {
+  render() {
+    return (
+      <div className="discovery-stream">
+        <div className="layout">
```

Comment:
Are we expecting non-layout things inside discovery-stream? I suppose it might be useful to have an extra wrapper ?

Reply:
I could combine these